config/config-mp/src/test/java/io/helidon/config/mp/MpConfigSourcesTest.java

```diff
@@ -99,6 +100,15 @@ void testHelidonLazy() {
         assertThat("exists called exactly once", lazy.exists.get(), is(1));
     }
 
+    @Test
+    void testMpConfigSourcesNullConfig() {
+        try {
```

Comment:
Please use `assertThrows(NullPointerException.class, () -> MpConfigSources.create((Config) null));` instead.
It returns the thrown exception, so you can assert the message as well.
Same for the second test in this PR.

(import: `import static org.junit.jupiter.api.Assertions.assertThrows;`)

Reply:
Done